index.html

```diff
@@ -7,10 +7,127 @@
+               
+            <img src="./design/menu-hamburger.svg" class="hamburger-icon" alt="menu-icon" onclick="openMenu()">
+            <div class="topnav" id="myLinks">
+                <ul>
+                                   
+                    <li><a href="#">HOME</a></li>
+                    <li><a href="#">CAKES</a></li>
+                    <li><a href="#">ORDERIN</a></li>
+                    <li><a href="#">LESSONS</a></li>
+                    <li><a href="#">ABOUT</a></li>
+                   
+                </ul>
+            </div>
+        </section>
+        </nav>
+      
+
+
+    </header>
+    <main>
+        <hr>
+        <section class="welcome"> 
+            <div class="welcome-text">
+                <h1>Welcome</h1>
+                <p>You are at the home where sweetness starts. We have a huge range of cakes you can enjoy. Just choose what you like and leave the rest to us. </p>
+            </div>  
+            <div class="welcome-deal-wrapper">
+                <img class="welcome-image" src="./design/image14.jpg" alt="Cake picture">
+                <div class="monthly-deals">
+                    <div class="birthday-deals">
+                        <p>Birthdays Deals</p>
+                        <img src="design/image13.jpg" alt="birthday-cake">
+                    </div>
+                    <div class="marriage deals">
+                        <p>Wedding Deals</p>
+                        <img src="./design/image13.jpg" alt="birthday-cake">
+                    </div>
+                </div>
+            </div>
+           
+        </section>
+        <hr>
+        <section class="products">
+            <div class="square-cakes">
+                <div class="square-cakes-wrapper">
+                   <p>Square Cakes</p>
+                    <div class="products-first-images" >
+                        <img src="./design/image6.jpg" alt="square cake">
+                        <img src="./design/image7.jpg" alt="square cake">
+                    </div> 
+                </div>
+                
+                <button class="button-buy">
+                    Buy Square Cake
+                </button>
+            </div>
+            <div class="rounded-cakes">
+                <div class="rounded-cakes-wrapper">
+                    <p>Rounded Cakes</p>
+                    <div class="products-second-images">
+                        <img src="./design/image8.jpg" alt="rounded cake">
+                        <img src="./design/image9.jpg"alt="rounded cake">
+                    </div>
+                </div>
+                
+                <button class="button-buy">
+                    Buy Rounded Cake
+                </button>
+
+            </div>
+        </section>
+        
+    </main>
+
+    <footer>
+        <section class="footer-heading">
+            <p>Join us on</p>
+        </section>
+        <section class="footer-icons">
+            <img src="./design/instagram-icon.svg">
```

Comment:
Make sure you add alt attributes on all images for accessibility reasons ;)

Reply:
thank you Makanti, I added alt attributes to the images